1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers= numbers.sort(); // complete this statement
```

Comment:
`.sort()` mutates the original array, so it's a good idea to create a copy of the array prior to sorting, for example:
`let sortedNumbers= [...numbers].sort()`, there is some more info on the spread operator here https://www.javascripttutorial.net/es6/javascript-spread/

Reply:
yeah. I have amended the code to avoid mutation. thanks.